services/accumulo-service/src/main/java/datawave/microservice/accumulo/lookup/LookupController.java

```diff
@@ -0,0 +1,59 @@
+package datawave.microservice.accumulo.lookup;
+
+import datawave.microservice.accumulo.lookup.LookupService.Parameter;
+import datawave.microservice.authorization.user.ProxiedUserDetails;
+import datawave.webservice.query.exception.QueryException;
+import datawave.webservice.response.LookupResponse;
+import io.swagger.annotations.ApiImplicitParam;
+import io.swagger.annotations.ApiImplicitParams;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.http.MediaType;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.annotation.security.RolesAllowed;
+
+import static datawave.microservice.accumulo.lookup.LookupService.ALLOWED_ENCODING;
+
+/**
+ * REST controller for Accumulo lookup service
+ */
+@RestController
+@ConditionalOnProperty(name = "accumulo.lookup.enabled", havingValue = "true")
+@RolesAllowed({"AuthorizedUser", "AuthorizedServer", "InternalUser", "Administrator"})
+@RequestMapping(path = "/v1", produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE})
+public class LookupController {
+    
+    @Autowired
+    private LookupService lookupService;
```

Comment:
It should be possible to make this final and set in the constructor with autowiring.

Reply:
> It should be possible to make this final and set in the constructor with autowiring.

Like [this](https://github.com/NationalSecurityAgency/datawave/blob/febd10fa76daa3d6ec7bdf485681973e3263ab0f/services/authorization-service/src/main/java/datawave/microservice/authorization/AuthorizationOperations.java#L37).